dev/io.openliberty.wsoc.internal_fat/fat/src/io/openliberty/wsoc/tests/SecureTest.java

```diff
@@ -66,7 +66,9 @@
 
     @BeforeClass
     public static void setUp() throws Exception {
-        LS.startServer();
+        // Test fail if running twice without building since app cannot start when validated
+        // Turn off validation, add the app if missing and then make the validation ourselves
+        LS.startServerAndValidate(true, true, false);
```

Comment:
I'm confused this this part. Can you elaborate. 

I don't understand why the test fails on the second run?  The check to add the add if it doesn't exist is already in place?

Reply:
Spoke with Issac: This encounter when testing locally (runFat).